### ISSUE TITLE
Missing delete orders in open orders pane RC1.1411

### ISSUE THREAD

**Issue by Bluffet** [bitshares/bitshares-ui#1607]
I tested out the release candidate RC1.1411 and found to my surprise the delete orders function was gone from the open orders pane. 

/Users/FunkIT/Dropbox/Screenshots/Screenshot 2018-06-19 14.07.55 copy.png

I know it's a prerelease, but I thought I'd mention it. 

Running it on a Mac OS 10.9.5

**Comment by Bluffet** [comment]
/Users/FunkIT/Dropbox/Screenshots/Screenshot.png

**Comment by Bluffet** [comment]
Fuck. Github don't like images.

**Comment by Bluffet** [comment]
Fuck. Github don't like images.

**Comment by Bluffet** [comment]
https://t.me/BitSharesDEX/295915

**Comment by clockworkgr** [comment]
Just copy the image (the image...not the filename) and paste in the textarea...

**Comment by clockworkgr** [comment]
![image](https://user-images.githubusercontent.com/6826762/41599884-137b82c2-73dd-11e8-9a29-a7fc846920c0.png)

**Comment by svk31** [comment]
Are you referring to 2.0.180629-rc1? I have no idea what RC1.1411 is...

**Comment by startailcoon** [comment]
Do you have the keys for the current account you're showing? Unless you have those, these components aren't visible as they have no function to you.

They are visible for my accounts on the 2.0.180629-rc1 version.

![image](https://user-images.githubusercontent.com/12114550/41615775-7072fd7e-73fc-11e8-94f3-ad4fd428ca40.png)

**Comment by abitmore** [comment]
@startailcoon your reply got the situation into scope of https://github.com/bitshares/bitshares-ui/issues/1285.

**Comment by Bluffet** [comment]
Correct, @svk. I got that version number from the about-box. 

@startailcoon, yes I have the keys.

**Comment by Bluffet** [comment]
Update: I have since managed to get the boxes back again by changing node. And I have managed to get them to disappear when connecting to the Nuremberg and  the Berlin node. 

Connecting to ihashfurys french node AND a restart fixes the issue. 

So far so good. I got rid of the orders I wanted. And am about place new ones. Which results in a NEW issue. I get a white screen when I confirm the new orders. It forces me to reload and for a brief moment I can see it reconnects to API server. The orders get placed, but I have to force a reload with a subsequent authentication. I tried this with several nodes with the same result. 

So to sum it up, there seems to be some things going on that involves connection. Or it might be the version of the node software for what I know. The white screen issue is something I have not seen before. But I'm persuing a possible workaround. 

I also looked in the dev tools for a possible explanation. The console had these messages:

![Screenshot 2018-06-20 03.59.15.png](https://cdn.steemitimages.com/DQmSbHtv6tW7uhAno9DYmBv71iKAJGDPrn4qFkkE3JKDkSN/Screenshot%202018-06-20%2003.59.15.png)

It might shed some light on it.

**Comment by Bluffet** [comment]
Update: I have since managed to get the boxes back again by changing node. And I have managed to get them to disappear when connecting to the Nuremberg and the Berlin node.

Connecting to ihashfurys french node AND a restart fixes the issue.

So far so good. I got rid of the orders I wanted. And am about place new ones. Which results in a NEW issue. I get a white screen when I confirm the new orders. It forces me to reload and for a brief moment I can see it reconnects to API server. The orders get placed, but I have to force a reload with a subsequent authentication. I tried this with several nodes with the same result.

So to sum it up, there seems to be some things going on that involves connection. Or it might be the version of the node software for what I know. The white screen issue is something I have not seen before. But I'm persuing a possible workaround.

I also looked in the dev tools for a possible explanation. The console had these messages:



It might shed some light on it.

**Comment by Bluffet** [comment]
![Screenshot 2018-06-20 03.59.15.png](https://cdn.steemitimages.com/DQmfMfowNecitVpiH581LkNSjtdXcbekx44YW4EAy2vqagU/Screenshot%202018-06-20%2003.59.15.png)

**Comment by Bluffet** [comment]
Again with a fresh restart and straight to placing an order:

![Screenshot 2018-06-20 09.12.41.png](https://cdn.steemitimages.com/DQmfS8JHsGYnkjsNhxpKmuZQFaxC7hxCajaW28xLNAZHjJW/Screenshot%202018-06-20%2009.12.41.png)

**Comment by Bluffet** [comment]
Result: white screen

**Comment by startailcoon** [comment]
Issue when placing orders is fixed with issue https://github.com/bitshares/bitshares-ui/issues/1605

**Comment by wmbutler** [comment]
@Bluffet please close if resolved.
